src/common.ts

```diff
@@ -0,0 +1,107 @@
+import { Address, BigInt, ethereum } from "@graphprotocol/graph-ts"
+import { ERC20 } from "../generated/UniswapV2Factory/ERC20"
+import {
+    Token,
+    RewardToken,
+    DexAmmProtocol,
+    LiquidityPool,
+} from "../generated/schema"
+import {
+    Network,
+    ProtocolType,
+    RewardTokenType,
+} from "./constants"
+
+const getUuid = require('uuid-by-string')
```

Comment:
"uuid-by-string" is a JS library without type declaration; is this the preferred way of importing such a library in TS?

Reply:
I'm not a TS expert but I think the more common way is something like `import getUuid from "uuid-by-string"`